package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "wpds-docs",
+  "name": "docs",
```

Comment:
is this an intentional change?

Reply:
this change came about when I ran `npm i`. Should I turn it back?